pkg/controller/masteruserrecord/sync.go

```diff
@@ -205,14 +207,33 @@ func (s *Synchronizer) openShift3XConsoleURL(apiEndpoint string) (string, error)
 	return url, nil
 }
 
+func (s *Synchronizer) alignDisabled() {
+	for _, uaStatus := range s.record.Status.UserAccounts {
+		if !isDisabled(uaStatus.Conditions) {
+			return
+		}
+	}
+	s.record.Status.Conditions, _ = condition.AddOrUpdateStatusConditions(s.record.Status.Conditions, toBeDisabled())
+}
+
+func isDisabled(conditions []toolchainv1alpha1.Condition) bool {
```

Comment:
This would be useful to have in common: https://github.com/codeready-toolchain/toolchain-common/blob/master/pkg/condition/condition.go
Can we introduce a new function there. Something like: 
```
// HasConditionReason returns true if the first Condition with given conditionType from the given slice has the specified reason
func HasConditionReason(conditions []toolchainv1alpha1.Condition, conditionType toolchainv1alpha1.ConditionType, reason string) bool {
	con, found := FindConditionByType(conditions, conditionType)
	return found && con.Reason == reason
}
```